### ISSUE TITLE
Elrond starts with only Onagers

### ISSUE THREAD

**Issue by BriceLeduc** [jj248/RealmsInExile#1053]
We need to assign him some actually useful MaA

**Comment by Jauffr** [comment]
We discussed this and agreed we would wait until Vector has implemented his concept of generic high elven MAA before doing any tweaking to Elrond in this regard.
I dont think you can assign MAA to people in any case though. If it was, that would be another way to resolve it though I suppose. I've just never seen any way to do it

**Comment by Lewa263** [comment]
I found something that might be relevant. In the `00_ai.txt` file in defines/ai, there are the following lines:
`MEN_AT_ARMS_REALM_SIZE_FOR_COST_EFFECTIVENESS_START = 5	# At realm sizes of this and below, men at arms quality is based on stat points compared to cost
MEN_AT_ARMS_REALM_SIZE_FOR_COST_EFFECTIVENESS_END = 50	# At realm sizes after this, men at arms quality is completely unaffected by cost. Between the two values, we interpolate between the unmodified and modified quality`
Imladris is under realm size 5 at start, so that will affect how the MaA are chosen.

I already demonstrated in the #beta-suggestions Discord channel that halving all Noldor MaA costs starts Elrond off with no onagers. I've also tried 60% cost, but that still causes a bunch of onagers. CK3 loads too slowly for me to work through all the percentages between 50 and 60.
